### DISCUSSION TITLE
Question regarding the LSP

### DISCUSSION THREAD

**Discussion by Unaimend** [quarto-dev/quarto-nvim#131] [Q&A]
Hello, for normal Python and R files I have pyright and r_languageserver installed and those work. But AFAIK those don't work out of the box with quarto. I had a quick look at the Kickstarter but not really understand how it works. 

So my question is: Is there any way LSP to use with quarto files so I can use my normal LSP keybindings to jump to definition etc. ?

**Comment by jmbuhr** [comment]
You'll have to pass your own keybindings to the config: https://github.com/quarto-dev/quarto-nvim?tab=readme-ov-file#configure
The reason for this is that the way we can handle languages in code chunks is via https://github.com/jmbuhr/otter.nvim
Though I do have plans to make manually configuring this obsolete in the future: https://github.com/jmbuhr/otter.nvim/issues/90
